src/test/java/lotto/WinningNumberTest.java

```diff
@@ -1,40 +1,87 @@
 package lotto;
 
-import lotto.domain.Lottos;
+import lotto.domain.LottoNumber;
+import lotto.domain.LottoTicket;
 import lotto.domain.WinningNumbers;
 import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
-import org.junit.jupiter.params.ParameterizedTest;
-import org.junit.jupiter.params.provider.ValueSource;
 
 import java.util.ArrayList;
+import java.util.List;
 
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.assertj.core.api.Assertions.assertThatThrownBy;
 
 public class WinningNumberTest {
 
+    @Test
+    @DisplayName("보너스 번호 매치 테스트")
+    public void bonusNumberMatch(){
+        WinningNumbers winningNumbers = new WinningNumbers(List.of(3,5,7,9,6,10),11);
+        assertThat(winningNumbers.bonusNumberMatch(
+                List.of(
+                        LottoNumber.of(3)
+                        ,LottoNumber.of(5)
+                        , LottoNumber.of(7)
+                        , LottoNumber.of(9)
+                        , LottoNumber.of(13)
+                        , LottoNumber.of(11)
+                )
+            )
+        ).isTrue();
+    }
 
-    @ParameterizedTest
-    @ValueSource(ints = {3,5,6,7,8,9})
-    @DisplayName("당첨번호 매치 테스트")
-    public void winningNumbersMatch(int value){
-        WinningNumbers winningNumbers = new WinningNumbers("3,5,6,7,8,9");
-        assertThat(winningNumbers.getWinningNumbers()).contains(value);
+    @Test
+    @DisplayName("보너스 번호 매치 실패 테스트")
+    public void bonusNumberFailMatch(){
+        WinningNumbers winningNumbers = new WinningNumbers(List.of(3,5,7,9,6,10),12);
+        assertThat(winningNumbers.bonusNumberMatch(
+                        List.of(
+                                LottoNumber.of(3)
+                                ,LottoNumber.of(5)
+                                , LottoNumber.of(7)
+                                , LottoNumber.of(9)
+                                , LottoNumber.of(13)
+                                , LottoNumber.of(11)
+                        )
+                )
+        ).isFalse();
+    }
+    @Test
+    @DisplayName("당첨번호 당첨 갯수 테스트")
+    public void winningNumbersMatch(){
+        WinningNumbers winningNumbers = new WinningNumbers(List.of(3,5,7,9,18,17),15);
+        assertThat(winningNumbers.lottoMatchCount(
+                List.of(
+                      LottoNumber.of(11)
+                    , LottoNumber.of(2)
+                    , LottoNumber.of(3)
+                    , LottoNumber.of(5)
+                    , LottoNumber.of(7)
+                    , LottoNumber.of(9)
```

Comment:
```suggestion
        assertThat(winningNumbers.lottoMatchCount(List.of(11, 2, 3, 5, 7, 9))
```
위와 같이 테스트 가능하도록 지원하는 것이 테스트 코드를 더 깔끔하게 유지 가능하지 않을까?